modules/claims_api/lib/claims_api/vbms_uploader.rb

```diff
@@ -59,8 +60,8 @@ def fetch_upload_token(filepath:, file_number:)
         file_number: file_number,
         va_receive_date: Time.zone.now,
         doc_type: @doc_type,
-        source: 'BVA',
-        subject: @doc_type,
+        source: 'LH-B', # proposed change from 'BVA',
```

Comment:
Looking back at this, I'm curious how VBMS is getting `lighthouse`. I wonder if they're using `BVA` and this could cause an issue?

Reply:
Yeah, I thought I would probably need to change it back, but pushed that up so we could have a reference when checking with other teams. TY for checking with Caracal!